lightningd/invoice.c

```diff
@@ -393,6 +397,10 @@ void invoice_try_pay(struct lightningd *ld,
 	payload->set = set;
 	tal_add_destructor2(set, invoice_payload_remove_set, payload);
 
+	/* We notify here to benefit from the payload and because the hook callback is
```

Comment:
Comment no longer quite matches?  This is not the hook callback anymore, right?

Reply:
Good point!